jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -175,6 +175,13 @@ private static void completeInitialization() {
 		if (openj9.internal.criu.InternalCRIUSupport.isCRaCSupportEnabled()) {
 			// export java.base/jdk.crac unconditionally
 			J9VMInternals.class.getModule().implAddExports("jdk.crac"); //$NON-NLS-1$
+
+			// export jdk.management/jdk.crac.management unconditionally
+			java.util.Optional<Module> om = ModuleLayer.boot().findModule("jdk.management");  //$NON-NLS-1$
+			if (om.isEmpty()) {
+				throw new InternalError("module jdk.management wasn't found"); //$NON-NLS-1$
```

Comment:
I don't think this should throw if `jdk.management` is not present; it seems reasonable to `jlink` a runtime without that module.

Reply:
This is guarded with `[IF CRAC_SUPPORT]` and `openj9.internal.criu.InternalCRIUSupport.isCRaCSupportEnabled()`, the target image generated by `jlink` will be required to include module `jdk.management` and others for `CRaC` functionalities.

Reply:
Neither that requirement nor the exception is necessary. Commenting it out and building `images` things work fine:
```sh
$ make images
$ cd images
$ ./jdk/bin/jlink --add-modules java.base,java.management --output=basic --no-header-files --no-man-pages
$ ./basic/bin/java -XX:CRaCCheckpointTo=crac --list-modules
java.base@21.0.4-internal
java.management@21.0.4-internal
```
The need for `jdk.management` should only be due to an application's need for that module, not just because the full jdk supports CRaC.

Reply:
I created https://github.com/eclipse-openj9/openj9/pull/19553 to stop throwing the exception.